pkg/application/doguInstallationUseCase.go

```diff
@@ -10,21 +11,22 @@ type DoguInstallationUseCase struct {
 	doguDomainUseCase domainservice.DoguInstallationDomainUseCase
 }
 
-func (useCase *DoguInstallationUseCase) validateDoguHealth() error {
-	installedDogus, err := useCase.doguRepo.GetAll()
+func (useCase *DoguInstallationUseCase) validateDoguHealth(ctx context.Context) error {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `(*DoguInstallationUseCase).validateDoguHealth` is unused

Reply:
This is a stub and is needed for the next feature.